Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -119,6 +119,9 @@ public final class DartAnalysisServerService implements Disposable {
   private static final long ANALYSIS_IN_TESTS_TIMEOUT = TimeUnit.SECONDS.toMillis(10);
   private static final long TESTS_TIMEOUT_COEFF = 10;
 
+  // LSP over Legacy Dart Analysis Protocol
+  public static final long LSP_MESSAGE_TEXT_DOCUMENT_CONTENT_TIMEOUT = TimeUnit.SECONDS.toMillis(100);
```

Comment:
Isn't it too much ;)?

Reply:
Reduced to `50`, `10` would probably be sufficient in most cases, just trying to be careful.  Remember, Dart is single-threaded and files could be large.